packages/fast-usdc-deploy/src/upgrade-evm-dests.core.js

```diff
@@ -37,14 +38,50 @@ const config = /** @type {const} */ ({
         version: 'ics20-1',
       },
     },
+    /**
+     * TODO: final values forthcoming
```

Comment:
I'll assume you have a process for ensuring these are updated before ship. Maybe FIXME?